cmd/syncthing/gui.go

```diff
@@ -350,14 +351,23 @@ func (s *apiService) Serve() {
 		// only set when run by the tests
 		close(s.started)
 	}
+
 	err := srv.Serve(listener)
 
 	// The return could be due to an intentional close. Wait for the stop
 	// signal before returning. IF there is no stop signal within a second, we
 	// assume it was unintentional and log the error before retrying.
 	select {
 	case <-s.stop:
-	case <-s.configChanged:
+	case listenAddress := <-s.listenAddressChanged:
+		l.Warnln("Listen address changed: ", listenAddress)
```

Comment:
So that one being a warning is probably fine, not sure if this one should be a warning though.